docs/PyCDE.md

```diff
@@ -43,9 +55,33 @@ $ cmake -G Ninja ../llvm/llvm \
     -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
     -DCIRCT_BINDINGS_PYTHON_ENABLED=ON \
     -DCIRCT_ENABLE_FRONTENDS=PyCDE
+    -DPython3_EXECUTABLE=<path_to_python3.6+> \
```

Comment:
PyCDE requires 3.8+ IIRC. Maybe 3.7. CMake doesn't check for it though.

Reply:
Oops, yes this is a typo on top of being unnecessary. I was having issues with CMake/Python not finding all the packages because we have multiple versions of Python3 installed. I was including the executable path to resolve that; I will remove it. (I guess this may be why people recommend python virtual envs?- maybe in the future we could look at creating a script that sets up a venv and installs all the requirements)

Reply:
> maybe in the future we could look at creating a script that sets up a venv and installs all the requirements

I didn't notice this thread when I posted earlier, but that is starting to sound a lot like Poetry :) It might make sense to document / recommend a Poetry based workflow, if we are getting to the point of automating the process of setting up a venv with the requirements.

Reply:
This is intended to be onboarding documentation -- similar to the MLIR onboarding docs. We should look at Poetry in the future.